js/qmwp-timeline.js

```diff
@@ -369,25 +262,25 @@ var AnalyzePage = function () {
     };
 
     var retrieveSettings = function () {
-        if (typeof(Storage) !== "undefined") {
-            dateSelectorVisible = (localStorage["dateSelectorVisible"] || "true") == "true" ? true : false;
-            inputSelectorVisible = (localStorage["inputSelectorVisible"] || "true") == "true" ? true : false;
-            outputSelectorVisible = (localStorage["outputSelectorVisible"] || "true") == "true" ? true : false;
+        if (typeof(Storage) !== 'undefined') {
+            dateSelectorVisible = (localStorage['dateSelectorVisible'] || 'true') == 'true' ? true : false;
+            inputSelectorVisible = (localStorage['inputSelectorVisible'] || 'true') == 'true' ? true : false;
+            outputSelectorVisible = (localStorage['outputSelectorVisible'] || 'true') == 'true' ? true : false;
```

Comment:
Line is too long.<br>['outputSelectorVisible'] is better written in dot notation.